src/Pages/NewArticlePage.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from "react";
+import Grid from "@material-ui/core/Grid";
+import AXIOS from "../utils/AXIOS";
+import { RouteComponentProps } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+import TagsInput from "../components/Article/TagsInput";
+import { withStyles, WithStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import { styles } from "./styles/NewArticlePageStyle";
+import { Formik, Form, FormikActions } from "formik";
+import * as yup from "yup";
+import FormikTextField from "../components/FormikInputs/FormikTextField";
+
+interface Values {
+  title: string;
+  description: string;
+  body: string;
+}
+
+const NewArticleSchema = yup.object().shape({
+  title: yup.string().required("Title can't be blank"),
+  description: yup.string().required("Description can't be blank"),
+  body: yup.string().required("body can't be blank")
+});
+
+const NewArticle: React.FC<
+  WithStyles<typeof styles> & RouteComponentProps<{ slug?: string }>
+> = props => {
+  const [articleTitle, setArticleTitle] = useState("");
+  const [articleDescription, setArticleDescription] = useState("");
+  const [articleBody, setArticleBody] = useState("");
+  const [tagsList, setTagsList] = useState([] as string[]);
```

Comment:
instead of `useState([] as string[])` write it as generic `useState<string[]>([])`